### PR TITLE
Platform changes for hawthorn theming

### PR DESCRIPTION
**Pull request by caesar2164**

### CODE REVIEW

File: lms/static/sass/multicourse/_course_about.scss [hunk 1]
```diff
@@ -636,4 +614,31 @@
   }
 }
 
+// Sneak-Peek button Styling
```
Comment (stvstnfrd) [inline lms/static/sass/multicourse/_course_about.scss]: Why not put this in the existing `../stanford/course/_sneakpeek_course_about.scss`?

File: lms/templates/header/header.html [hunk 2]
```diff
@@ -45,7 +45,7 @@
             <span class="line"></span>
             <span class="line"></span>
         </div>
-        % if user.is_authenticated:
+        % if UserProfile.has_registered(user):
```
Comment (stvstnfrd) [inline lms/templates/header/header.html:49]: Good catch.

**Review by stvstnfrd — CHANGES_REQUESTED** [review]

File: lms/static/sass/multicourse/_course_about.scss (hunk 1 above)
Comment (caesar2164) [reply lms/static/sass/multicourse/_course_about.scss]: good point

File: lms/static/sass/multicourse/_course_about.scss [hunk 3]
```diff
@@ -164,28 +164,6 @@
             }
           }
 
-          span.register, span.add-to-cart, span.sneakpeek {
```
Comment (stvstnfrd) [inline lms/static/sass/multicourse/_course_about.scss:167]: Is `add-to-cart` old or new shopping cart?

How does this file `diff` against upstream now?

File: lms/templates/header/header.html [hunk 4]
```diff
@@ -14,7 +14,7 @@
 # App that handles subdomain specific branding
 from branding import api as branding_api
 from openedx.core.djangoapps.lang_pref.api import header_language_selector_is_enabled, released_languages
-
+from student.models import UserProfile
```
Comment (stvstnfrd) [inline lms/templates/header/header.html:17]: Nit, but can we leave the newline at the bottom? below this import?

It'll shrink the diff ;)

**Review by stvstnfrd — APPROVED** [review]

**Comment by stvstnfrd** [comment]
cherry-picked..

**Pull request closed without merging**